src/test/llmnode_test.cpp

```diff
@@ -2687,13 +2685,14 @@ TEST_F(EmbeddingsHttpTest, simplePositiveBase64) {
         handler->dispatchToProcessor(endpointEmbeddings, requestBody, &response, comp, responseComponents, &writer),
         ovms::StatusCode::OK);
     rapidjson::Document d;
-    d.Parse(response.c_str());
+    rapidjson::ParseResult ok = d.Parse(response.c_str());
+    ASSERT_EQ(ok.Code(), 0);
     ASSERT_EQ(d["object"], "list");
-    ASSERT_EQ(d["data"].IsArray(), true);
+    ASSERT_TRUE(d["data"].IsArray());
     ASSERT_EQ(d["data"].Size(), 1);
     ASSERT_EQ(d["data"][0]["object"], "embedding");
-    ASSERT_EQ(d["data"][0]["embedding"].IsString(), true);
-    ASSERT_EQ(d["data"][0]["embedding"].Size(), ((4 * (EMBEDDING_OUTPUT_SIZE * sizeof(float)) / 3) + 3) & ~3);
+    ASSERT_TRUE(d["data"][0]["embedding"].IsString());
+    ASSERT_EQ(d["data"][0]["embedding"].Size(), ((4 * (EMBEDDING_OUTPUT_SIZE * sizeof(float)) / 3) + 3) & ~3); // In base64 each symbol represents 3/4 of a byte rounded up
```

Comment:
Why is `& ~3` needed? Can we use round from STL?

Reply:
std::round would round to the nearest integer, not the next symbol